src/helpers/slack.ts

```diff
@@ -0,0 +1,109 @@
+const { Reaction, ErrorCode } = require('@slack/web-api');
+import SLACK_MESSAGE_BLOCKS from './files/welcomeMessageBlocks.json';
+import { SlackEvent, SlackWebClient } from './types';
+const fs = require('fs');
+
+
+function getEmojisToReactWith(text: string): Array<string> {
```

Comment:
so this is function only TS no "class" as such, interesting.

Reply:
I am quite curious on this one. When would you recommend making use of class in Javascript/TS? I keep hearing to have everything in functional programming style for JS.

Reply:
Again a module with functions != functional programming. Functional programming is a different paradigm with usually pure functions.